app/src/main/java/com/zulip/android/activities/MessageListFragment.java

```diff
@@ -206,11 +206,39 @@ public boolean onContextItemSelected(MenuItem item) {
                 copyMessage(message);
                 Toast.makeText(getContext(), R.string.message_copied, Toast.LENGTH_SHORT).show();
                 return true;
+            case R.id.star_message:
+                starMessage(message);
+                Toast.makeText(getContext(), R.string.message_starred, Toast.LENGTH_SHORT).show();
+                return true;
+            case R.id.un_star_message:
+                unStarMessage(message);
+                Toast.makeText(getContext(), R.string.message_un_starred, Toast.LENGTH_SHORT).show();
+                return true;
             default:
                 return super.onContextItemSelected(item);
         }
     }
 
+    /**
+     * Stars a message passed as parameter
+     *
+     * @param message Message to be starred
+     */
+    private void starMessage(Message message) {
+        message.setMessageStar(true);
+        recyclerView.getAdapter().notifyDataSetChanged();
+    }
+
+    /**
+     * Unstars a message passed as parameter
+     *
```

Comment:
I used default reformatting by pressing `Ctrl` + `Shift` + `Alt` + `L` as @kunall17 suggested to refromat the code that we send (after merge of bc5a25f).

@Sam1301 Shall I remove all those extra spaces added by auto-formatting?